docs/cookbook/birthdayIn2030.mjs

```diff
@@ -1,6 +1,6 @@
 const birthday = Temporal.MonthDay.from('12-15');
 
-const birthdayIn2030 = birthday.withYear(2030);
+const birthdayIn2030 = birthday.toDate(2030);
```

Comment:
Hmm, here is an example of where I really prefer the old names ... `toDate(year)` as below is OK, but `toDate(2030)` with a number literal is more mystifying to me. I know I was in the minority on that viewpoint so I definitely don't want to push for bringing the old names back; but maybe there is a third way? How about calling it toDateIn()? e.g. `toDateIn(2030)`, `toDateIn({ era: 'reiwa',  year: 12 })`